core/txpool/blobpool/blobpool.go

```diff
@@ -334,9 +334,11 @@ func New(config Config, chain BlockChain) *BlobPool {
 	}
 }
 
-func (p *BlobPool) SetAstriaOrdered(types.Transactions) {}
-func (p *BlobPool) ClearAstriaOrdered()                 {}
-func (p *BlobPool) AstriaOrdered() *types.Transactions  { return &types.Transactions{} }
+func (p *BlobPool) SetAstriaOrdered(types.Transactions)              {}
+func (p *BlobPool) ClearAstriaOrdered()                              {}
+func (p *BlobPool) UpdateAstriaExcludedFromBlock(*types.Transaction) {}
+func (p *BlobPool) AstriaExcludedFromBlock() *types.Transactions     { return &types.Transactions{} }
```

Comment:
Seems like this public function is added solely for test purposes? Is there another way to manage that?

Reply:
let me see

Reply:
don't think there is, we can't attribute functions with a `#cfg[test]` like how we do in rust :( , we need to do hacks like https://github.com/golang/go/blob/master/src/reflect/export_test.go

There are some cases in the Geth codebase where they have written public functions only for testing like this one:
https://github.com/astriaorg/astria-geth/blob/a9326e133d39265dada6fae5c73f9a24ed70d563/miner/payload_building.go#L144